mmv1/products/securityposture/posture.yaml

```diff
@@ -0,0 +1,519 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Posture'
+description: |
+  A Posture represents a collection of policy set including its name, state, description
+  and policy sets. A policy set includes set of policies along with their definition.
+  A posture can be created at the organization level.
+  Every update to a deployed posture creates a new posture revision with an updated revision_id.
+import_format: ['{{%parent}}/locations/{{location}}/postures/{{posture_id}}']
+base_url: '{{parent}}/locations/{{location}}/postures'
+
+self_link: '{{parent}}/locations/{{location}}/postures/{{posture_id}}'
+
+create_url: '{{parent}}/locations/{{location}}/postures?postureId={{posture_id}}'
+
+update_url: '{{parent}}/locations/{{location}}/postures/{{posture_id}}?revisionId={{revision_id}}'
```

Comment:
Do we provide the current `revision_id` to update, effectively a custom etag? https://google.aip.dev/134#etags

Reply:
Oh, there's also an etag- I'm not sure what this does. Is there a user guide or reference doc you can share?